gemrb/plugins/SDLVideo/SDLVideo.cpp

```diff
@@ -228,6 +228,9 @@ int SDLVideoDriver::ProcessEvent(const SDL_Event & event)
 				EvntManager->DispatchEvent(std::move(e));
 			}
 			break;
+#else
+		default:
+			break;
 #endif
```

Comment:
both cases, whether the define is set or not, need the default.

Reply:
The default case wasn’t there originally, so I figured you only wanted it when USE_SDL_CONTROLLER_API is undefined. 

But no worries, I’ll have to do it tomorrow.

Reply:
It's just MISRA/MITRE/CERT defensive programming best practice. It clearly shows that we indeed thought about all the possible cases the value can take.